### DISCUSSION TITLE
How to search for timestamps with ranges (prev. "ts inside drawers")

### DISCUSSION THREAD

**Discussion by karthink** [alphapapa/org-ql#402] [General]
I have Org headings with contents like the following:

```
* 127 Meeting
:PROPERTIES:
:ETag:     "3411048842930000"
:org-gcal-managed: gcal
:END:
:org-gcal:
<2024-01-26 Fri 09:00-12:00>
:END:
```

So a properties block (not germane to this discussion), and a drawer with an active timestamp.

When I search for entries using `ts-active:to=today` or `(ts-active :to today)`, no entries like the above are discovered.

1. Am I using the query syntax wrong?
2. If not, does org-ql ignore timestamps that are inside drawers in the entry?
3. Is there a way to find these entries with org-ql-search?

**Comment by alphapapa** [comment]
That timestamp is a range, which isn't currently supported.  It doesn't matter that it's in a drawer.  Please see https://github.com/alphapapa/org-ql/issues/371

**Reply by karthink** [reply]
Thanks for the clarification @alphapapa.
